p2p/src/network.rs

```diff
@@ -69,8 +69,8 @@ where
     pub broker: Broker,
     /// Flag that stops listening stream
     finish_sender: Option<Sender<()>>,
-    /// Mailbox capacity
-    mailbox: u32,
+    /// Actor channel capacity
```

Comment:
how do you measure capacity? the number of blocks? I think this should be more descriptive than just repeating the `actor_channel_capacity` without underscores